lib/inspec/runner.rb

```diff
@@ -182,58 +182,10 @@ def register_rules(ctx)
 
     private
 
-    def block_source_info(block)
```

Comment:
This just moved over to the rspec_formatter because I think it makes more sense there.  If we want I can do that in a different pull request.